router/batchrouter/asyncdestinationmanager/bing-ads/offline-conversions/bulk_uploader.go

```diff
@@ -66,6 +66,17 @@ func (b *BingAdsBulkUploader) Transform(job *jobsdb.JobT) (string, error) {
 			return payload, err
 		}
 	}
+
+	// Validate conversionTime format as ISO 8601 (RFC3339)
+	// Bing ads supports timestamp of the format 2020-01-01T00:00:00.10000Z
+	conversionTimeStr, ok := fields["conversionTime"].(string)
+	if !ok {
+		return payload, fmt.Errorf("conversionTime field is not a string")
+	}
+	if _, err := time.Parse(time.RFC3339, conversionTimeStr); err != nil {
+		return payload, fmt.Errorf("conversionTime must be in ISO 8601 format like 2006-01-02T15:04:05Z07:00")
```

Comment:
It's different from what was mentioned on our doc [here](https://www.rudderstack.com/docs/destinations/reverse-etl-destinations/bing-ads-offline-conversions/connect-retl-source/#create). Are we going to update our doc?

Reply:
Yes, will get doc team to work on it